docs/README.rst

```diff
@@ -201,67 +213,86 @@ sslProfile           JSON object       Optional                   BIG-IP SSL pro
-+---------------------+-----------+-----------+-----------+-------------------------------------------------------+---------------------------+
-| iappVariables       | key-value | Required  |           | Define the iApp variables                             |                           |
-|                     | object    |           |           | needed for service creation.                          |                           |
-+---------------------+-----------+-----------+-----------+-------------------------------------------------------+---------------------------+
+The ``frontend.virtualServer`` properties provide the information required to deploy an iApp on the BIG-IP system.
+
+.. tip::
+
+   The ``iappOptions`` represent information that the user would be prompted to provide if deploying the iApp via the BIG-IP configuration utility.
+
+\
+
+==================== ================= ============== ======================================================= ====================================
+Property             Type              Required       Description                                             Allowed Values
+==================== ================= ============== ======================================================= ====================================
+partition            string            Required       The BIG-IP partition you want the |kctlr| to manage.                                            
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iapp                 string            Required       BIG-IP iApp template to use to create the               Any iApp template that already 
+                                                      application  Service.                                   exists on the BIG-IP system. 
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iappPoolMemberTable  JSON object       Required       Define the name and layout of the pool member table   
+                                                      in the iApp.                                          
+                                    
+                                                      **See** :ref:`iApp Pool Member Table`.         
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iappTables           JSON object       Optional       Define iApp tables to apply to the Application Service
+                     array
+                                                      Example: ::
+                                                      
+                                                      "iappTables": {                           
```

Comment:
This is not rendering well in your staged docs.

Reply:
fixed